src/Finch_SolidificationData.hpp

```diff
@@ -266,6 +266,74 @@ class SolidificationData
 
         fout.close();
     }
+
+    std::vector<double> getBounds()
```

Comment:
Will ExaCA need this on device? Even if not, `Kokkos::Array` is more flexible in that way

Reply:
Sounds good, I don't need them on the device as of now (just used to set up the grid struct in ExaCA) but I'll make that change